### PR TITLE
UserGuide: Add Find (by other fields), Find (by tag) and Create group…

### PR DESCRIPTION
**Pull request by sofarsophie**
Updated commands for V2.0

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -212,6 +272,46 @@ The `redo` command fails as there are no `undo` commands executed previously.
...
+=== [Coming in v2.0] Creating groups : `create`
+
+Creates a group and adds contacts, identified by indices from the last persons listing, to the group.
+Format: `create GROUP_NAME [INDEX] [MORE_INDICES]...`
+
+***
+* An empty group with no persons can be created.
+***
+
+=== [Coming in v2.0] Creating a new humanDEX : `create_humanDEX`
```
Comment (kengying) [inline docs/UserGuide.adoc]: new humanDEX, may be better? to lessen the confusion between creating group and create new humanDEX

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -212,6 +272,46 @@ The `redo` command fails as there are no `undo` commands executed previously.
 `redo` (reapplies the `clear` command) +
 // end::undoredo[]
 
+=== [Coming in v2.0] Creating groups : `create`
```
Comment (kengying) [inline docs/UserGuide.adoc]: isn't this to be done by 1.5?

Comment (sofarsophie) [reply docs/UserGuide.adoc]: updated!

File: docs/UserGuide.adoc (hunk 1 above)
Comment (sofarsophie) [reply docs/UserGuide.adoc]: updated!

**Pull request merged**